docs/hipify-clang.md

```diff
@@ -7,7 +7,7 @@ After applying all the matchers, the output HIP source is produced.
 **Advantages:**
 
 1. It is a translator; thus, any even very complicated constructs will be parsed successfully, or an error will be reported.

```

Comment:
Recommended

>It is a translator; thus, any even very complicated constructs will be parsed successfully, or an error will be reported.

1. `hipify-clang` is a translator; any complex constructs will be parsed successfully, or an error will be reported.

Reply:
I will consider your recommendations in the upcoming separate doc changes. This one is about formatting only.